.github/workflows/ci_web.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+      - name: corepack
+        run: corepack enable
```

Comment:
Looks like `corepack` is required to upgrade yarn. 
![Uploading Screenshot 2024-07-22 at 7.15.49 AM.png…]()